tests/check/check_h26xsigned_auth.c

```diff
@@ -1625,38 +1628,65 @@ START_TEST(vendor_axis_communications_operation)
 END_TEST
 #endif
 
-/* Test description */
-START_TEST(public_key_on_validation_side_from_start)
+static signed_video_t *
+generate_and_set_private_key_on_camera_side(struct sv_setting settings, bool add_public_key_to_sei, nalu_list_item_t *i_nalu, nalu_list_item_t **sei)
```

Comment:
settings -> setting, since it is one single setting and not an array of settings